components/dashboard/src/service/public-api.ts

```diff
@@ -4,46 +4,16 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import { createConnectTransport, createPromiseClient, Interceptor } from "@bufbuild/connect-web";
+import { createConnectTransport, createPromiseClient } from "@bufbuild/connect-web";
 import { Team as ProtocolTeam } from "@gitpod/gitpod-protocol/lib/teams-projects-protocol";
 import { TeamsService } from "@gitpod/public-api/lib/gitpod/experimental/v1/teams_connectweb";
 import { Team } from "@gitpod/public-api/lib/gitpod/experimental/v1/teams_pb";
 import { TeamMemberInfo, TeamMemberRole } from "@gitpod/gitpod-protocol";
 import { TeamMember, TeamRole } from "@gitpod/public-api/lib/gitpod/experimental/v1/teams_pb";
-import { getGitpodService } from "./service";
-
-let token: string | undefined;
-
-const authInterceptor: Interceptor = (next) => async (req) => {
-    if (!token) {
-        const newToken = await getGitpodService().server.generateNewGitpodToken({
-            type: 1,
-            scopes: [
-                "function:getGitpodTokenScopes",
-
-                "function:getWorkspace",
-                "function:getWorkspaces",
-
-                "function:createTeam",
-                "function:joinTeam",
-                "function:getTeams",
-                "function:getTeam",
-                "function:getTeamMembers",
-                "function:getGenericInvite",
-
-                "resource:default",
-            ],
-        });
-        token = newToken;
-    }
-
-    req.header.set("Authorization", `Bearer ${token}`);
-    return await next(req);
-};
 
 const transport = createConnectTransport({
     baseUrl: `${window.location.protocol}//api.${window.location.host}`,
-    interceptors: [authInterceptor],
+    credentials: "include",
```

Comment:
I assume all usage is featureFlagged up until now. Correct?

Reply:
It still remains feature flagged. This just changes the "transport" layer such that the calls use cookies instead of generating a token.

Reply:
Basically, this transport doesn't have any effect unless a call is actually made